test/e2e/dra/dra.go

```diff
@@ -2319,10 +2499,28 @@ func (b *builder) className() string {
 
 // class returns the device class that the builder's other objects
 // reference.
-func (b *builder) class() *resourceapi.DeviceClass {
+func (b *builder) class(suffix string) *resourceapi.DeviceClass {
+	ern := extendedResourceName
+	switch suffix {
+	case "":
+	case "2":
+		ern = extendedResourceName2
+	case "3":
+		ern = extendedResourceName3
+	case "4":
+		ern = extendedResourceName4
+	case "5":
+		ern = extendedResourceName5
+	case "6":
+		ern = extendedResourceName6
+	}
+
 	class := &resourceapi.DeviceClass{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: b.className(),
+			Name: b.className() + suffix,
+		},
+		Spec: resourceapi.DeviceClassSpec{
+			ExtendedResourceName: &ern,
```

Comment:
Let's avoid setting `ExtendedResourceName` in tests where it is not used. By setting it in all tests, we loose coverage for the "DeviceClass has no ExtendedResourceName" variant.

Reply:
done